lib/smart_answer_flows/am-i-getting-minimum-wage/questions/past_payment_date.govspeak.erb

```diff
@@ -3,6 +3,7 @@
 <% end %>
 
 <% options(
+  "2018-04-01": "Apr 2018 - Mar 2019",
```

Comment:
The messaging here says "You can go back up to 6 years", but it looks like you can actually go back 9 years.

Reply:
There's 9 years in the file but on the smart answer it only shows 6.

Reply:
Gotcha